src/v2/providers/identity.ts

```diff
@@ -20,17 +20,15 @@ interface InternalOptions {
 }
 
 /**
- * All function options plus idToken, accessToken, and refreshToken.
+ * @beta All function options plus idToken, accessToken, and refreshToken.
  */
 export interface BlockingOptions extends options.GlobalOptions {
   idToken?: boolean;
   accessToken?: boolean;
   refreshToken?: boolean;
 }
 
-/**
- * Handle an event that is triggered before a user is created.
- */
+/** @beta Handle an event that is triggered before a user is created. */
```

Comment:
we should yank beta per our offline discussion

Reply:
Done!